zello-sdk/src/main/java/com/zello/sdk/headset/HeadsetMediaSessionImpl.kt

```diff
@@ -39,8 +39,9 @@ class HeadsetMediaSessionImpl : HeadsetMediaSession {
 			val mediaButtonIntent = Intent(Intent.ACTION_MEDIA_BUTTON).run {
 				setClass(context.applicationContext, MediaButtonReceiver::class.java)
 			}
-			session.setMediaButtonReceiver(PendingIntent.getBroadcast(context.applicationContext, 0, mediaButtonIntent, 0))
-			session.isActive = true;
+			val intentFlag = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) PendingIntent.FLAG_IMMUTABLE else 0
```

Comment:
It's Android M, not S, that introduced `FLAG_IMMUTABLE`. Ultimately, we should bump the min version to M anyway, which will remove the need to conditionally apply the flag.
```suggestion
			val intentFlag = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) PendingIntent.FLAG_IMMUTABLE else 0
```